src/tests/libecalc/integration/snapshots/test_all_consumer_with_time_slots_models/test_all_consumer_with_time_slots_models_results/all_consumer_with_time_slots_models_v3.json

```diff
@@ -654,8 +671,8 @@
                                 ]
                             },
                             "is_valid": [
-                                false,
-                                false
+                                true,
```

Comment:
The compressor stage operates within its capacity so this individual compressor now has is_valid true. The comparison to target pressures has moved to the compressor train - is_valid is still false on the compressor train.